_protocols/data-analysis.md

```diff
@@ -0,0 +1,113 @@
+BeerDecoded-Nanopore data analysis protocol
+
+
+# Software installation
+
+
+## MinKnow
+1. Follow the instructions here to install MinKnow. MinKnow is the interface software to the Minion handheld device, for configurations, sequencing initiation and transfer of the sequencing signals from the device to the computer. 
```

Comment:
We do have build a container to make this simpler isn't it ... maybe we need to link it here?